cmd/controller-init/main.go

```diff
@@ -57,15 +63,62 @@ func main() {
 		logger.Fatalw("Failed to get cluster config", zap.Error(err))
 	}
 
+	// Check into the config-secrets. Its *possible* that the ConfigMap
```

Comment:
nit: I think it would be worth throwing in a huge heading showing that the code in this section is for the v0.3 upgrade (and maybe enclosing it in scope brackets)

that way as we go along we know when everything was added

Reply:
I'll add the scope brackets and a note mentioning this is for upgrading to v0.3 and beyond.